src/Makefile

```diff
@@ -5576,7 +5630,7 @@ sms_tiny:
 	-DLESS_TEXT \
 	-DNO_PRINT \
 	-create-app \
-	-DBUFFRED \
+	-DBUFFERD \

```

Comment:
Still a typo. :smiling_imp:

Reply:
This is now fixed.
My Makefile needs a huge clean-up.